tedana/viz.py

```diff
@@ -95,18 +131,19 @@ def write_comp_figs(ts, mask, comptable, mmix, n_vols,
             seconds_val = round(X * tr, 2)
             ax2Xs.append(seconds_val)
         ax_ts2.set_xticks(ax1Xs)
-        ax_ts2.set_xbound(ax_ts.get_xbound())
+        ax_ts2.set_xlim(ax_ts.get_xbound())
         ax_ts2.set_xticklabels(ax2Xs)
         ax_ts2.set_xlabel('seconds')
 
         ax_ts.plot(mmix[:, compnum], color=line_color)
 
         # Title will include variance from comptable
-        comp_var = "{0:.2f}".format(comptable.iloc[compnum][3])
-        comp_kappa = "{0:.2f}".format(comptable.iloc[compnum][1])
-        comp_rho = "{0:.2f}".format(comptable.iloc[compnum][2])
-        plt_title = 'Comp. {}: variance: {}%, kappa: {}, rho: {}'.format(compnum, comp_var,
-                                                                         comp_kappa, comp_rho)
+        comp_var = "{0:.2f}".format(comptable.iloc[compnum]["variance explained"])
```

Comment:
Should we use "variance explained" or "normalized variance explained" here and elsewhere?
@emdupre Thoughts?

Reply:
Thinking more on this, both of these sum up to be close to 100, even in cases where the PCs entered into in the ICA step explain <100% of the variance, right, because ICs are not orthogonal? I'm looking at an older run of the meica tedana, where the comp_table says 76.7% of the variance was explained, but summing the Variance and Variance norm columns gives me 100.03 and 96.78% respectively. 

Means that `100 - sum(variance_expl)` that calculates the unexplained variance is never going to work quite right. 

Nor would calculating the variance explained (as is done in io.py, around line 202) and adding that in give the right number - because then the pie chart would be relative to the total (ie ~100 variance from ICA + unexplained = some_number >100). Perhaps some scaling would be needed for the values, prior to giving them to that part of figure creation.

Reply:
:+1: for postponing the discussion until #223  !